lab5.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+#include "lab5.h"
+
+int main()
+{
+    TimeConverter timeconverter;
```

Comment:
the names of the objects are written in capital letters according to convention. Use 'TimeConverter'.